i18n: Add full support (#399)
pydata/pydata-sphinx-theme: docs/user_guide/i18n.rst

Internationalization
====================

> 

Reviewer: Perhaps we should split this into two sections. One for theme built-in strings, and another for user configurable strings.

Author: The first paragraph is built-in strings, the rest is user-configurable strings.

This theme contains translatable strings. Most strings will automatically be translated if the language is `supported <https://github.com/pydata/pydata-sphinx-theme/tree/master/pydata_sphinx_theme/locale>`__. To add another language, see :ref:`translating-the-theme`.

> The theme's :doc:`options<configuring>` also contain translatable strings, which require a different process. The following instructions assume that you store your translations in a ``locale`` directory under your documentation directory, and that you want to use ``theme`` as the name the message catalog for these strings.

Reviewer: ```suggestion
The theme's :doc:`options<configuring>` also contain translatable strings, which require a different process. The following instructions assume that you store your translations in a ``locale`` directory under your documentation directory, and that you want to use ``theme`` as the name of the message catalog for these strings.
```

> The theme's :doc:`options<configuring>` also contain translatable strings, which require a different process. The following instructions assume that you store your translations in a ``locale`` directory under your documentation directory, and that you want to use ``theme`` as the name the message catalog for these strings.

Reviewer: We should link to a useful introduction for Sphinx translation, in my experience most users don’t know how this works so I think this would helo

Author: I don't know that any are especially helpful. See e.g. https://www.sphinx-doc.org/en/master/usage/advanced/intl.html

> This theme contains translatable strings. Most strings will automatically be translated if the language is `supported <https://github.com/pydata/pydata-sphinx-theme/tree/master/pydata_sphinx_theme/locale>`__. To add another language, see :ref:`translating-the-theme`.
> 
> The theme's :doc:`options<configuring>` also contain translatable strings, which require a different process. The following instructions assume that you store your translations in a ``locale`` directory under your documentation directory, and that you want to use ``theme`` as the name of the message catalog for these strings.

Reviewer: ```suggestion
This theme contains translatable strings. There are two kinds of strings contained in this theme, with slightly different steps to translate each.

**Built-in strings** are hard-coded in the theme's templates, and will automatically be translated if the language is `supported <https://github.com/pydata/pydata-sphinx-theme/tree/master/pydata_sphinx_theme/locale>`__. To add another language, see :ref:`translating-the-theme`.

**Configurable strings** are user-defined in ``html_theme_options`` (see :doc:`options<configuring>` for examples). To translate these strings, see the section below.

Translating configurable strings
--------------------------------

These are instructions for translating configurable strings (those that are customizable in ``html_theme_options``).
The instructions assume that you store your translations in a ``locale`` directory under your documentation directory, and that you want to use ``theme`` as the name of the message catalog for these strings.
```

#. In your ``conf.py`` file:

>    .. code-block:: rst

Reviewer: ```suggestion
   .. code-block:: python
```
